packages/excalidraw/fractionalIndex.ts

```diff
@@ -0,0 +1,256 @@
+import { mutateElement } from "./element/mutateElement";
+import { ExcalidrawElement } from "./element/types";
+import {
+  generateKeyBetween as _generateKeyBetween,
+  generateJitteredKeyBetween as _generateJitteredKeyBetween,
+  generateNKeysBetween as _generateNKeysBetween,
+  generateNJitteredKeysBetween as _generateNJitteredKeysBetween,
+  base62CharSet as _base62CharSet,
+  indexCharacterSet,
+} from "fractional-indexing-jittered";
+import { ENV } from "./constants";
+import { InvalidFractionalIndexError } from "./errors";
+
+type FractionalIndex = ExcalidrawElement["index"];
+
+export const base36CharSet = indexCharacterSet({
+  chars: "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+  firstPositive: "A",
+  mostPositive: "Z",
+});
+
+const { chars, firstPositive, mostPositive, mostNegative } = _base62CharSet();
+
+const base62CharSet = indexCharacterSet({
+  chars,
+  firstPositive,
+  mostPositive,
+  mostNegative,
+});
```

Comment:
do we need to re-instantiate this now that we're not modifying the jitter range?

Reply:
it's not necessary, will remove it

Reply:
removed